create-custom-map-style/tileserver/downloadSpritesGlyphs.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+#!/bin/bash
+
+mkdir sprites
+mkdir glyphs
+cd sprites
+aws location get-map-sprites --file-name sprites.png --map-name $1 --profile $2 sprites.png &
```

Comment:
Why background (`&`) these commands?

Reply:
My shell scripting skills aren't the greatest, but the idea is to not have the output from the AWS CLI block your terminal . We could also pipe the output into a file, but either way I want to leave the terminal open for the user to be able to keep typing commands.

Reply:
Ah, ok. These commands should run quickly enough that you'll achieve that without backgrounding.

Reply:
Ok I'll remove them. I think most people will know how to ctrl + c out of it.